### PR TITLE
Try to use ES6 imports instead of CJS

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
We're experimenting here to see if we can use ES6 imports in a Github Action.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/153378

### Tests
This PR _is_ the test 🤯

I tested this by taking away most of the functionality of the `bumpVersion` action (just fetch tags using the github client sdk and then console.log the results) and adding it to the `test` workflow so it triggered whenever I pushed code to this PR. Since that worked with ES6 module imports, it's safe to assume that this will work just the same. But the only way to know for sure is to merge this PR.

### Tested On
- Github

### CODE REVIEW

**Review by chiragsalian — APPROVED** [review]
I think it looks good to me but I'll tag @marcaaron for another set of eyes.

**Comment by marcaaron** [comment]
I'm not sure I see the clear value in this change and don't necessarily agree with the premise in the linked issue. 

That said, I also don't see the harm in experimenting with ESM in the github actions and also barely understand how github actions work tbh. 

@AndrewGable thoughts?

**Comment by AndrewGable** [comment]
I am a bit confused, why can't we test this before merging by just changing the trigger to push?

**Comment by roryabraham** [comment]
@AndrewGable I did do some testing. Checkout [this workflow run](https://github.com/Expensify/Expensify.cash/runs/1826324574?check_suite_focus=true), where I checkout [this commit](https://github.com/Expensify/Expensify.cash/pull/1396/files/77f88b4a059392a972d6a7156f54909eca4d2f60 from this branch and successfully run ES6 code in a JS Github Action. I should have included that in the description, but technically speaking it isn't a perfect test but should give us some confidence that this will work.

**Pull request closed without merging**
